tests/fxl-layout_duplication/OPS/page_001.xhtml

```diff
@@ -0,0 +1,11 @@
+<html xmlns="http://www.w3.org/1999/xhtml" xmlns:epub="http://www.idpf.org/2007/ops" xml:lang="en">
+  <head>
+    <title>rendition:layout duplication</title>
+    <meta name="viewport" content="width=900, height=600" />
+    <link rel="stylesheet" href="fixed.css" />
+  </head>
+  <body id="p1">
+    <h1>rendition:layout duplication</h1>
+    <p>Page 1. This test passes if EPUBCheck returns an error.</p>
```

Comment:
Just making a bookmark: if we decide something on what the RS should do, this statement should be extended...